src/modules/map/components/LocationPopup.vue

```diff
@@ -148,7 +148,7 @@ async function copyShareLink() {
 }
 
 function clearClick() {
-    store.dispatch('clearClick')
+    store.dispatch('clearClick', { dispatcher: 'LocationPopup.vue' })
```

Comment:
even though there's only one action being used, might be a good idea to declare a const at the top of the file anyway (so that we stick to do the same logic-ish across all files)

Reply:
done